templates/zerver/for/working-groups-and-communities.md

```diff
@@ -0,0 +1,37 @@
+Working groups and communities need different things out of chat than
```

Comment:
Maybe something more like "Part-time organizations like learning communities, standards bodies, advocacy groups, and hobby groups need different things out of chat than companies do, since while there may be some core members spending many hours per week on the organization, much of the value of the community comes from large numbers of members or potential members who have an unrelated job and thus likely have less than an hour a week to spend on the organization."

We also might want this page to link over to the open source one, maybe with an introduction mentioning that if the community is a technical group that might discuss programming, one should check out the page for open source projects.